github/enterprise_manage_ghes_config.go

```diff
@@ -59,8 +59,8 @@ type ConfigApplyEventsNodeEvent struct {
 	ConfigRunID  *string    `json:"config_run_id,omitempty"`
 	TraceID      *string    `json:"trace_id,omitempty"`
 	SpanID       *string    `json:"span_id,omitempty"`
-	SpanParentID *int       `json:"span_parent_id,omitempty"`
-	SpanDepth    *int       `json:"span_depth,omitempty"`
+	SpanParentID *int64     `json:"span_parent_id,omitempty"`
+	SpanDepth    *int64     `json:"span_depth,omitempty"`
```

Comment:
Sorry I wasn't clear.  This field does not need to be `*int64`... it should remain `*int`.
Same situation for all fields that are "integers" but are _not_ "IDs".

Please revert all `int64` field changes _unless_ it is an "ID".